misc/gencxxtypedumper.sh

```diff
@@ -36,18 +36,25 @@ echo "	vStringCatS (buf, str);"
 echo "	return true;"
 echo "}"
 echo
+echo "static vString * g_pDebugBuffer = NULL;"
+echo
 echo 'const char * cxxDebugTypeDecode (enum CXXTokenType eType)'
 echo '{'
 echo '	bool a = false;'
-echo '	static vString *buf;'
-echo '	buf = vStringNewOrClear (buf);'
```

Comment:
If I can remember what I did correctly, all your changes in this commit are not needed.
Instead of calling `vStringNewOrClear`, call `vStringNewOrClearWithAutoRelease`.
See processLangKindDefinition() in main/parse.c as example code using `vStringNewOrClearWithAutoRelease`.